src/main/resources/templates/gcc/activities-self-employment.html

```diff
@@ -1,28 +1,13 @@
-<!DOCTYPE html>
-<html th:lang="${#locale.language}" xmlns:th="http://www.thymeleaf.org">
-<head th:replace="~{fragments/head :: head(title='TEMPLATE')}"></head>
-<body>
-<div class="page-wrapper">
-  <div th:replace="~{fragments/toolbar :: toolbar}"></div>
-  <section class="slab">
-    <div class="grid">
-      <div th:replace="~{fragments/goBack :: goBackLink}"></div>
-      <main id="content" role="main" class="form-card spacing-above-35">
-        <th:block th:replace="~{fragments/cardHeader :: cardHeader(header=#{placeholder}, subtext=#{placeholder})}"/>
-        <th:block th:replace="~{fragments/form :: form(action=${formAction}, content=~{::formContent})}">
-          <th:block th:ref="formContent">
-            <div class="form-card__content">
-                <!-- Put inputs here -->
-            </div>
-            <div class="form-card__footer">
-              <th:block th:replace="~{fragments/inputs/submitButton :: submitButton(text=#{general.inputs.continue})}"/>
-            </div>
-          </th:block>
-        </th:block>
-      </main>
-    </div>
-  </section>
-</div>
-<th:block th:replace="~{fragments/footer :: footer}"/>
-</body>
-</html>
\ No newline at end of file
+<th:block
+  th:replace="~{fragments/screens/screenWithYesAndNoButtons ::
+  screenWithYesAndNoButtons(
+    title=#{activities-self-employment.title},
+    header=#{activities-self-employment.header},
+    subtext=#{activities-self-employment.subtext},
+    iconFragment=~{fragments/gcc-icons :: baby},
+    formAction=${formAction},
+    inputName='isSelfEmployed',
+    ariaLabel='header',
+    inputContent=~{::inputContent})}">
+  <th:block th:ref="inputContent"/>
```

Comment:
We want the company name header here as well
```suggestion
  <th:block th:ref="inputContent">
    <th:block th:replace="~{fragments/inputs/text ::
        text(inputName='companyName',
        ariaLabel='header')}"/>
    </th:block>
</th:block>
```